client/analytics/components/report-chart/index.js

```diff
@@ -222,6 +228,7 @@ export default compose(
 
 		if ( query.search && ! ( query[ endpoint ] && query[ endpoint ].length ) ) {
```

Comment:
Check if the value is an array

```suggestion
		if ( query.search && ! ( Array.isArray( query[ endpoint ] ) && query[ endpoint ].length ) ) {
```

Reply:
I think this is always returned as a comma delimited string, but if I'm wrong we'll also need to update this in all other areas.

Reply:
Right, like @joshuatf said, `query[ endpoint ]` is a comma-separated string.